extensions/amp-autocomplete/0.1/test/test-amp-autocomplete.js

```diff
@@ -23,26 +24,367 @@ describes.realWin('amp-autocomplete', {
   },
 }, env => {
 
-  let win;
-  let element;
+  let win, doc;
 
-  beforeEach(() => {
-    win = env.win;
-    element = win.document.createElement('amp-autocomplete');
-    win.document.body.appendChild(element);
-    toggleExperiment(win, 'amp-autocomplete', true);
-  });
+  describe('test extension', () => {
+    beforeEach(() => {
+      win = env.win;
+      doc = win.document;
+      toggleExperiment(win, 'amp-autocomplete', true);
+    });
 
-  it('should have hello world when built with experiment on', () => {
-    element.build();
-    expect(element.querySelector('div').textContent).to.equal('hello world');
-  });
+    function getAutocomplete(attributes,
+      json = '{ "items" : ["apple", "banana", "orange"] }') {
+      win.sessionStorage.clear();
```

Comment:
Is this `win.sessionStorage` line needed for your test?

Reply:
Don't think so--good catch!